HISTORY.md

```diff
@@ -1,5 +1,8 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+
+  * Update outdated codes in cf.md and datasetmapper.md (#3398).
```

Comment:
```suggestion
  * Update outdated code in tutorials (#3398).
```
Sorry to be picky, but many people who read this changelog will have no context on what those files are, so, better to try and keep things general. :+1: